### ISSUE TITLE
Show pronouns next to display name

### ISSUE THREAD

**Issue by thibaultmol** [tuskyapp/Tusky#4238]
Someone noticed that Moshidon (and potentially Megalodon) clients look if there is a custom field on the profile of a user called 'pronouns', if so. Displays those next to the display in the timeline
https://yiff.life/@ThatGeekZen/111800176000218255
Not officially spec'ed but could be could a nice addition to Tusky as well

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by nehemiagurl** [comment]
to my knowledge FediLab also does this as well.

**Comment by thibaultmol** [comment]
Correct, I also asked them and they implemented it a couple days later but only landed in stable this week

**Comment by achaviara** [comment]
This issue seems to be implemented. Pronoun field is retrieved and displayed. It is not next to the name but it can be found in the section below, where all fields are displayed.

**Comment by russell-phillips** [comment]
> This issue seems to be implemented. Pronoun field is retrieved and displayed. It is not next to the name but it can be found in the section below, where all fields are displayed.

It's displayed when viewing a profile, but the request is for the pronoun field to be displayed _in the timeline_.
